app/assets/javascript/method.js

```diff
@@ -212,9 +212,15 @@ window.method = {
 
   setIndex: function (index) {
     storeData.method_index = index;
+    document.getElementById("force-refresh-method").click();
```

Comment:
@mgiammar Why do you need to trigger a force refresh?

Reply:
@deepanshs It's needed so when a new method is selected (in any tab) the spectrum updates. The previous code relied on similar logic but within clientside_callbacks but didn't work with my updated JavaScript. This just simplifies updating which method is selected.